src/views/Home/Home.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import NasaCard from '../../components/NasaCard/NasaCard';
+import { getPictureBySearch, getPictureOfTheDay } from '../../services/fetchdata';
+import Controls from '../../components/Controls/Controls';
+
+export default function Home() {
+  const [nasaData, setNasaData] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [dates, setDates] = useState();
+  const [tempStartDate, setTempStartDate] = useState();
+  const [tempEndDate, setTempEndDate] = useState();
+
+  useEffect(() => {
```

Comment:
I might break this up into two useEffects. One to handle the initial loading of data, one that handles fetching by date